### PR TITLE
ZPRP-10-dodac-endpoint-do-dodawania-warunkow-zaliczen

### PR DESCRIPTION
**Pull request by kubao503**

### CODE REVIEW

File: studsched/app/api/base.py [hunk 1]
```diff
@@ -56,3 +56,14 @@ async def subjects() -> Any:
...
+
+
+@base_router.post("/subjects/{subject_id}/requirements")
+async def add_requirements(
+    requirements: list[Requirement],
+    subject_id: int,
+) -> list[Requirement]:
+    """Adds new requirement for a given subject"""
+
+    # Do some db staff
```
[the remark this answers is not in the document]
Comment (kubao503) [reply studsched/app/api/base.py:68]: Not by me 😁

[the remark this answers is not in the document]
Comment (kozlowskimaciej) [reply studsched/app/api/base.py:68]: By your fingers

File: studsched/app/schemas/base.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ class VersionResponse(BaseModel):
 class TaskType(IntEnum):
     LAB = auto()
     EXAM = auto()
-    KOL = auto()
+    COLL = auto()
```
Comment (kozlowskimaciej) [inline studsched/app/schemas/base.py:18]: There were more types than that , make the list complete

Comment (kozlowskimaciej) [reply studsched/app/schemas/base.py:18]: And don’t use abbreviations

**Review by kozlowskimaciej — CHANGES_REQUESTED** [review]

File: studsched/app/schemas/base.py (hunk 2 above)
Comment (kubao503) [reply studsched/app/schemas/base.py:18]: Where were those types? What are you talking about?

Comment (kubao503) [reply studsched/app/schemas/base.py:18]: I want to see them

Comment (kozlowskimaciej) [reply studsched/app/schemas/base.py:18]: Use your imagination

**Review by kozlowskimaciej — APPROVED** [review]

**Pull request merged**